.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
```

Comment:
This can be set to true, it just puts a small link at the bottom of the picker, fairly low abuse potential I think 

![image](https://github.com/pingdotgg/uploadthing/assets/11494384/5da312f4-7110-4cbf-b0bf-65d2ec586473)